sklearn/covariance/_graph_lasso.py

```diff
@@ -588,9 +589,28 @@ class GraphicalLassoCV(GraphicalLasso):
     cv_alphas_ : list of shape (n_alphas,), dtype=float
         All penalization parameters explored.
 
+        .. deprecated:: 0.23
+            The `cv_alphas_` attribute is deprecated in version 0.23 in favor
+            of `cv_results_['alphas']` and will be removed in version 0.25
+
     grid_scores_ : ndarray of shape (n_alphas, n_folds)
         Log-likelihood score on left-out data across folds.
 
+        .. deprecated:: 0.23
+            The `grid_scores_` attribute is deprecated in version 0.23 in favor
+            of `cv_results_` and will be removed in version 0.25
+
+    cv_results_ : dict of numpy ndarrays
+        Dictinoary, with attributes:
+
+        alphas : ndarray of shape (n_alphas,)
```

Comment:
we have attributes without underscore?

Reply:
I think since these are members of the `cv_results_`, they don't need the underscore, do they?

Reply:
I incorrectly wrote "attributes" in the description for `cv_results_`. This has been updated.